libs/platform/experience/src/models/composition.model.ts

```diff
@@ -4,6 +4,12 @@ import { LayoutAttributes } from '../../layout/src';
 
 export interface CompositionProperties {
   rules?: StyleRuleSet[];
+  visibility?: ComponentVisibility;
+}
+
+export interface ComponentVisibility {
+  hide?: boolean;
+  hideRule?: string;
```

Comment:
what about dropping `hide` and use a hideRule for that? `hideAlways` or smth

Reply:
The best that I came up with: `hide` and `hideByRule`